common/src/main/scala/org/apache/celeborn/common/util/PbSerDeUtils.scala

```diff
@@ -457,6 +459,22 @@ object PbSerDeUtils {
     builder.build()
   }
 
+  def fromPbApplicationInfo(pbApplicationInfo: PbApplicationInfo): ApplicationInfo = {
+    val applicationInfo = new ApplicationInfo()
+    applicationInfo.updateFileCount(pbApplicationInfo.getFileCount)
+    applicationInfo.updateTotalWritten(pbApplicationInfo.getTotalWritten)
+    applicationInfo.setHeartbeatTime(pbApplicationInfo.getLastHeartbeatTime)
+    applicationInfo
+  }
+
+  def toPbApplicationInfo(applicationInfo: ApplicationInfo): PbApplicationInfo = {
```

Comment:
PbApplicationMeta need add userIdentifier and should serialize/deserialize to/from ratis log.